src/pages/Main/MainProduct/MainProduct.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const MAIN_PRODUCT = [
+  {
+    id: 1,
+    img: 'product_img_url',
+    brand: 'Nike',
+    product: 'Dunk Low Wolf Grey Rosewood',
+    price: 140000,
+  },
+  {
+    id: 2,
+    img: 'product_img_url',
+    brand: 'Jordan',
+    product: 'Jordan 1 x Travis Scott Retro Low OG SP Black Phantom',
+    price: 990000,
+  },
+  {
+    id: 3,
+    img: 'product_img_url',
+    brand: 'IAB',
+    product: 'IAB Studio Pigment Sweatshirt & Mini Bag Black',
+    price: 139000,
+  },
+  {
+    id: 4,
+    img: 'product_img_url',
+    brand: 'The North Face',
+    product: 'The North Face 1996 Eco Nuptse Jacket Black',
+    price: 275000,
+  },
+];
+
+const MainProduct = () => {
+  return (
+    <>
+      <MainProductList>
+        {MAIN_PRODUCT.map(({ id, img, brand, product, price }) => {
+          const productPrice = price
+            .toString()
+            .replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
3자리 마다 , 붙히는 로직이군요! 
toLocaleString을 사용하면 쉽게 구현 가능합니다.

```suggestion
          const productPrice = price.toLocaleString();
```

Reply:
넵 감사합니다 :) 반영해서 수정완료 했습니다 !!